ebpf/aya-ebpf/src/args.rs

```diff
@@ -168,6 +173,36 @@ impl<T> FromPtRegs for *const T {
     }
 }
 
+#[cfg(bpf_target_arch = "powerpc64")]
+impl<T> FromPtRegs for *const T {
+    fn from_argument(ctx: &pt_regs, n: usize) -> Option<Self> {
+        if n <= 7 {
+            unsafe { bpf_probe_read(&ctx.gpr[3 + n]).map(|v| v as *const _).ok() }
+        } else {
+            None
+        }
+    }
+
+    fn from_retval(ctx: &pt_regs) -> Option<Self> {
+        unsafe { bpf_probe_read(&ctx.gpr[3]).map(|v| v as *const _).ok() }
+    }
```

Comment:
Just to verify that I did this section correctly. For `from_retval()` for PowerPC, I used https://github.com/libbpf/libbpf/blob/master/src/bpf_tracing.h#L170 as reference and returning the value of `__PT_RC_REG`.

I wasn't sure about this. X86 and ARM64 are returning `__PT_RC_REG`, but RISCV appears to be returning `__PT_RET_REG`. Is RISCV coded wrong or is there something I'm missing?
X86: https://github.com/libbpf/libbpf/blob/master/src/bpf_tracing.h#L150
ARM64: https://github.com/libbpf/libbpf/blob/master/src/bpf_tracing.h#L250
RISCV: https://github.com/libbpf/libbpf/blob/master/src/bpf_tracing.h#L374

Reply:
Also LGTM